CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 All notable changes to this project will be documented in this file.
 
-## v0.1.13 - NOT RELEASED
+## v0.2.0 - 2019-11-18
 #### Upgrade Notes:
   * This version we recommend upgrading the Postgres database to 11.5.  Before you run the upgrade it is recommended that you go into the AWS console and update the version to 10 and choose apply immediately.  Then do it again and update to 11.5.
   * After the upgrade a reindex will be required.  Before you reindex we will need to you connect to the platform postgres system and run the following.
```

Comment:
Can you make it more explicit that the reindex is to be run after:
* the database has been upgraded to 11.5
* terraform apply has been run to deploy the new version

Reply:
Just pushed a change.  Let me know if that makes more sense.